src/index.html

```diff
@@ -10,10 +10,41 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="header">
+    <a href="#" class="logo"><img src="images/logo.png" alt="logo"></a>
```

Comment:
place `<img>` on the next line and don't forget about placing nested tags on the new line